include/readdy/model/actions/Actions.h

```diff
@@ -352,11 +352,30 @@ class BreakBonds : public TimeStepDependentAction {
  */
 class EvaluateCompartments : public Action {
 public:
-    explicit EvaluateCompartments() : Action() {}
+    EvaluateCompartments() : Action() {}
 
     ~EvaluateCompartments() override = default;
 };
 
+class InitializeKernel : public Action {
+public:
+    InitializeKernel() : Action() {}
+};
+
+/* Not an Action, because perform needs TimeStep t. */
+class EvaluateObservables {
+public:
+    virtual void perform(TimeStep t) = 0;
+    virtual ~EvaluateObservables() = default;
+};
+
+/* Not an Action, because perform needs TimeStep t. */
```

Comment:
Doesn't the state model carry the time step? Then this could be a proper action.